addons/website_sale/models/res_partner.py

```diff
@@ -8,7 +8,11 @@
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    last_website_so_id = fields.Many2one('sale.order', compute='_compute_last_website_so_id', string='Last Online Sales Order')
+    last_website_so_id = fields.Many2one(
+        comodel_name='sale.order',
+        compute='_compute_last_website_so_id',
+        string="Last Online Sales Order",
+    )
```

Comment:
```suggestion
    last_website_so_id = fields.Many2one(
        comodel_name='sale.order',
        string="Last Online Sales Order",
        compute='_compute_last_website_so_id',
    )
```
We could be more coherent with the order in our field's declaration IMHO, you sometime changed the order, but not always

Reply:
We decided on an order with ANV, now defined in our team guidelines.